ui/app/routes/vault/cluster/settings/auth/configure/section.js

```diff
@@ -41,6 +42,10 @@ export default Route.extend(UnloadModelRoute, {
     const { method } = this.paramsFor('vault.cluster.settings.auth.configure');
     const backend = this.modelFor('vault.cluster.settings.auth.configure');
     const modelType = this.modelType(backend.type, section_name);
+    if (getHelpUrlForModel(modelType, 'test')) {
```

Comment:
is "test" supposed to be hardcoded arg?

Reply:
It actually doesn't have to be anything, I'm using this function as a test to see if the given model has a helpUrl defined (basically the same as previously checking model.useOpenApi). I'll remove it and add a comment to what this is doing.